app-frontend/packs/components/core/UI/UiButton.vue

```diff
@@ -10,8 +11,11 @@ export default {
     name: 'UiButton',
     methods: {
         clickUiButton() {
-            this.$emit('clickUiButton')
+            this.$emit('click');
         }
+    },
+    props: {
+        typeButton:String,
```

Comment:
тоже касается и пропса. Компонент и так называется Button. В названии пропса нет необходимости писать `Button`, так как и так понятно. 
Также хорошей практикой является передавать пропс обьектом с полем required, чтобы избежать возможных ошибок 

Например в твоем случае это будет выглядеть так:
```suggestion
        type: {
            type: String,
            required: true
        },
```

Reply:
спасибо, учту